<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title></title>
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width">

        <link rel="stylesheet" href="css/bootstrap.min.css">
        <style>
            body {
                padding-top: 60px;
                padding-bottom: 40px;
            }
        </style>
        <link rel="stylesheet" href="css/bootstrap-responsive.min.css">
        <link rel="stylesheet" href="css/main.css">

        <script src="js/vendor/modernizr-2.6.2-respond-1.1.0.min.js"></script>
    </head>
    <body class="preview" id="top" data-spy="scroll" data-target=".subnav" data-offset="80">
        <!--[if lt IE 7]>
            <p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
        <![endif]-->

        <!-- This code is taken from http://twitter.github.com/bootstrap/examples/hero.html -->

        <div class="navbar navbar-fixed-top">
            <div class="navbar-inner">
                <div class="container">
                
                    <a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                    </a>
                    <a class="brand" href="#">&nbsp;</a>
                    <div class="nav-collapse collapse">
                        <ul class="nav">
                            <li class="active"><a href="#">Home</a></li>
                            <!--
                            <li><a href="#aboutxx">About</a></li>
                            <li><a href="#contact">Contact</a></li>
                            -->
                        </ul>
                    </div>
            
                </div>
               
            </div>
        </div>

        <div class="container">

            <!-- Main hero unit for a primary marketing message or call to action -->
            <!--
            <div class="hero-unit">
                <h1>Hello, world!</h1>
                <p>This is a template for a simple marketing or informational website. It includes a large callout called the hero unit and three supporting pieces of content. Use it as a starting point to create something more unique.</p>
                <p><a class="btn btn-primary btn-large">Learn more &raquo;</a></p>
            </div>
            -->
            <!-- Masthead
================================================== -->
<header class="jumbotron subhead" id="overview">
  <div class="row">
    <div class="span12">
      <br/>
      <h2>The 3rd International Workshop on Semantics-Powered Data Analytics (SEPDA 2018)</h2>
      <br/>
    <p class="lead">
        in conjunction with the <a href="http://orienta.ugr.es/bibm2018/" target="_blank">IEEE International Conference on Bioinformatics and Biomedicine</a> (BIBM 2018)<br/>
       to be held in Madrid, Spain<br/>
        December 3 - 6, 2018
        </p>
    </div>
    <div class="span6">
    <!--
      <div class="bsa well">
          <div id="bsap_1277971" class="bsarocks bsap_c466df00a3cd5ee8568b5c4983b6bb19"></div>
      </div>
    -->
    </div>
  </div>
  <div class="subnav">
    <ul class="nav nav-pills">
      <li><a href="#overview">Overview</a></li>
      <li><a href="#scope">Scope & Important Dates & Submission</a></li>
      <li><a href="#organization">Committees</a></li>
      <li><a href="#review">Review</a></li>
      <li><a href="#program">Program</a></li>
      <li><a href="#papers">Accepted</a></li>
      <li><a href="#registration">Registration</a></li>
      <li><a href="#contact">Contact</a></li>
      <li><a href="#history">Past Workshops</a></li>
    </ul>
  </div>
</header>

    <section id="overview">
      <div class="row">
        <div class="span12">
          <h3>Overview</h3>
          <p>Biomedical ontologies and controlled terminologies provide structured domain knowledge to a variety of health information systems. The rich thesaurus with concepts linked by semantic relationships has been widely used in natural language processing, data mining, machine learning, semantic annotation, and automated reasoning. The dramatically increasing amount of health-related data poses unprecedented opportunities for mining previously unknown knowledge with semantics-powered data analytics methods. However, due to the heterogeneity of different data sources, it is a challenging problem to exploit multiple sources to solve real-world problems such as designing cost-effective treatment plan for patients, designing generalizable clinical trials, drug repurposing, and clinical phenotyping. The goal of this workshop is to bring people in the field of knowledge representation, knowledge management, and health data analytics to discuss innovative semantic methods, applications, and data analytics to address problems in healthcare, biomedicine, public health, and clinical research with biomedical, clinical, behavioral, and social web data.</p>


          <p>We are inviting original research submissions (FULL 8 pages), work-in-progress (SHORT 4 pages), and poster abstracts (2 pages, NEW TYPE).</p>

          <p>All the accepted submissions will be presented in SEPDA 2018 and published in the IEEE BIBM 2018 Proceedings (in IEEE Xplore Digital Library); Selected FULL/SHORT papers will be invited to publish an extended version in the supplement of BMC Medical Informatics & Decision Making (IF: 2.134; Indexed in SCIE, MEDLINE, and PMC; and subject to an article processing charge of $1,465). Selected high-quality SHORT papers will also be invited to submit an extended version to the journal supplement for consideration.</p>
 
          <p>This is the third installment of the SEPDA workshop. The first SEPDA workshop (<a href="http://semantics-powered.org/sepda2016.html" target="_blank">SEPDA 2016</a>) was held in conjunction with IEEE BIBM 2016 in Shenzhen, China. SPEDA 2016 was a half-day workshop. Dr. Hua Xu, a leading expert in the field of Biomedical Informatics, gave the keynote speech on the bioCADDIE project. Eight quality papers were presented. Selected papers were invited to publish the extended version to the special issue on “Semantics-Powered Healthcare Engineering and Data Analytics” in the Journal of Healthcare Engineering (https://www.hindawi.com/journals/jhe/si/461480/).</p>
          <p>The second SEPDA Workshop (<a href="http://semantics-powered.org/sepda2017.html" target="_blank">SEPDA 2017</a>) was held in conjunction with IEEE BIBM 2017 in Kansas City, Missouri, USA. SPEDA 2017 was a full day workshop. 17 high quality papers were presented in the workshop. Extended journal version of the papers will be published in BMC Medical Informatics & Decision Making (IF: 1.6).</p>

          <p>The second SEPDA Workshop (SEPDA 2017) was held in conjunction with IEEE BIBM 2017 in Kansas City, Missouri, USA. SPEDA 2017 was a full day workshop. 17 high quality papers were presented in the workshop. 13 extended papers have been published in BMC Medical Informatics & Decision Making (IF: 2.134).</p>
        </div>
      </div>
    </section>

    <section id="scope">
      <div class="row">
        <div class="span12">
          <h3>Scope</h3>
          <p>We are inviting original research submissions as well as work-in-progress.</p>
          <p>Topics of interest include but not limited to:</p>
          <ul>
            <li><b>Ontologies and Controlled Terminologies</b>
              <ul>
                <li>Ontology development and enrichment</li>
                <li>Quality assurance of ontologies and controlled terminologies</li>
                <li>Semantic harmonization and ontology alignment</li>
                <li>Knowledge representation and reasoning</li>
              </ul>
            </li>
            <li><b>Semantics-based Data Analytics</b>
              <ul>
                <li>Ontology-based text mining and natural language processing</li>
                <li>Ontology-based analysis on biomedical, clinical, or social web data</li>
                <li>Information Extraction on biomedical, clinical, or social web data</li>
                <li>Data mining or machine learning on biomedical, clinical or social web data</li>
                <li>Semantic annotation on biomedical, clinical or social web data</li>
              </ul>
            </li>
            <li><b>Data Integration</b>
              <ul>
                <li>Linked open data</li>
                <li>Novel approaches for data integration of heterogenous data sources</li>
                <li>Large scale data integration</li>
              </ul>
            </li>
            <li><b>Application</b>
              <ul>
                <li>Novel tools and ontologies for data interpretation and visualization</li>
                <li>Pharmacovigilance and drug repurposing using ontologies</li>
                <li>Clinical trial generalizability assessment using ontologies</li>
                <li>Algorithmic phenotyping and cohort identification using ontologies</li>
                <li>Improving the literacy of health information consumers</li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
      <div class="row">
        <div class="span12">
          <h3>Important Dates</h3>
          <p>Sept 30, 2018: Due date for full / work-in-progress workshop papers submission</p>
          <p>Oct 27, 2018: Notification of paper acceptance to authors</p>
          <p>Nov 15, 2018: Camera-ready of accepted papers</p>
          <p>Dec 3, 2017: SEPDA 2018 Workshop</p>
          <p>Journal submission deadline: <b>TBD</b></p>
        </div>
      </div>
       <div class="row">
        <div class="span12">
          <h3>Submission</h3>
          <p>Please submit a full-length paper (up to 8 page IEEE 2-column format) through the online submission system (you can download the format instruction <a href="http://www.ieee.org/conferences_events/conferences/publishing/templates.html" target="_blank">here</a> ). Electronic submissions (in PDF or Postscript format) are required. Selected participants will be asked to submit their revised papers in a format to be specified at the time of acceptance.</p>
          <p>Submission site: <a href="https://wi-lab.com/cyberchair/2017/bibm17/scripts/submit.php?subarea=S03&undisplay_detail=1&wh=/cyberchair/2017/bibm17/scripts/ws_submit.php" target="_blank">Submit a New Paper</a> (https://wi-lab.com/cyberchair/2017/bibm17/scripts/submit.php?subarea=S03&undisplay_detail=1&wh=/cyberchair/2017/bibm17/scripts/ws_submit.php)</p>
        </div>
      </div>
    </section>


  <section id="organization">
      <div class="row">
        <div class="span12">
          <h3>Organizing</h3>
          <ul>
            <li><b>Organizing Committee</b>
              <ul>
                <li><a href="https://sites.google.com/site/henryhezhe2003/home" target="_blank">Zhe He</a>, School of Information, Florida State University, USA</li>
                <li><a href="https://sbmi.uth.edu/faculty-and-staff/cui-tao.htm" target="_blank">Cui Tao</a>, School of Biomedical Informatics, University of Texas Health Center at Houston, USA</li>
                <li><a href="https://zhangruisite.wordpress.com/" target="_blank">Rui Zhang</a>, Institute for Health Informatics, University of Minnesota, USA</li>
                <li><a href="http://jiangbian.me" target="_blank">Jiang Bian</a>, Health Outcomes & Biomedical Informatics, College of Medicine, University of Florida, USA</li>
              </ul>
            </li>
            <li><b>Program Committee</b>
              <ul>
                <!-- <li>William Hogan, University of Florida, USA</li> -->
                <li>GQ Zhang, University of Kentucky, USA</li>
                <!--<li>James Geller, New Jersey Institute of Technology, USA</li>-->
                <li>Guoqian Jiang, Mayo Clinic, USA</li>
                <!--<li>Zhihui Luo, University of Wisconsin, Milwaukee, USA</li>-->
                <li>Jake Luo, University of Wisconsin, Milwaukee, USA</li>
                <!--<li>Tianyong Hao, The cisco School of Informatics, Guangdong University of Foreign Studies, China</li>-->
                <!--<li>Amanda Hicks, University of Florida, USA</li>-->
                <!-- <li>Hong Cui, University of Arizona, USA</li> -->
                <!--<li>Yongqun “Oliver” He, University of Michigan, USA</li>-->
                <!--<li>Fernando Martin-Sanchez, Weill Cornell College of Medicine, USA</li>-->
                <!-- <li>Vasant Honavar, Pennsylvania State University, USA</li> -->
                <li>James Cimino, University of Alabama at Birmingham, USA</li>
                <li>Licong Cui, University of Kentucky, USA</li>
                <li>Xia Jing, Ohio University, USA</li>
                <!-- <li>Hongfang Liu, Mayo Clinic, USA</li> -->
                <li>Riccardo Miotto, Icahn School of Medicine at Mount Sinai, USA</li>
                <!--<li>Olivier Bodenreider, US National Library of Medicine, USA</li>-->
               <!--  <li>Dingcheng Li, Mayo Clinic, USA</li>
                <li>Kin Wah Fung,  US National Library of Medicine, USA</li> -->
                <li>Fleur Mougin, University of Bordeaux, France</li>
                <!--<li>Praveen Chandar, IBM Research, USA</li>-->
                <!-- <li>Samson Tu, Stanford University, USA</li> -->
                <!-- <li>Kevin Cohen, University of Colorado, USA</li> -->
               <!--  <li>Hua Xu, University of Texas Health Center at Houston, USA</li> -->
                <li>Yonghui Wu, University of Florida, USA</li>
                <!--<li>Ronald Cornet, University of Amsterdam, Netherlands</li>-->
                <li>Yi Guo, University of Florida</li>
                <!--<li>Juan Antonio Lossio-Ventura, University of Florida</li>-->
                <li>Buzhou Tang, Intelligent Computing Research Center, Harbin Institute of Technology Shenzhen, China </li>
                <li>Zhengxing Huang, Zhejiang University, China</li>
                <li>Feichen Shen, Mayo Clinic</li>
                <!--<li>Gang Feng, Northwestern University</li>-->
                <!--<li>Hua Li, School of Medicine, Washington University</li>
                <li>Han Wu, Rutgers University</li>
                <li>Yang Xiang, University of Texas Health Science Center at Houston</li>
                <li>Ryan Benton, University of South Alabama</li>
                <li>Ankur Agrawal, Manhattan College</li>-->
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </section>

    <section id="review">
      <div class="row">
        <div class="span12">
<!--           <h3>Review Process</h3>
          <p><b>Introduction to the new peer-review system – Pevals (Participant-engaged evaluation system)</b></p>
          <p>The review system we will use for this workshop is similar to the one NSF used in their 2013 pilot program to test an alternative review system (<a href="https://www.nsf.gov/pubs/2013/nsf13096/nsf13096.jsp" target="_blank">https://www.nsf.gov/pubs/2013/nsf13096/nsf13096.jsp</a>). Notable changes include using less reviewers for each paper and adding a discussion period before reviewers submitting the final scores.</p>
          <p>The review process consists of the following steps:</p>
          <p><b>Step 1</b>: Each contributing author will be assigned to review and rank two or four other papers. Review assignments will be made so as to avoid organizational or individual conflicts-of-interest.</p>
          <p><b>Step 2</b>: All authors must complete their first round of review and ranking of the assigned papers within 10 days of the date of their assignment. </p>
          <p><b>Step 3</b>: The authors will then have an opportunity to write rebuttals for the reviewers’ comments made to their papers and discuss with the anonymous reviewers about the comments and their replies. This step will take about three days.</p> 
          <p><b>Step 4</b>: Each author will then re-evaluate the papers assigned to them based on the new information collected in the discussion period and update their scores and rankings. This step will take another three days. </p>
          <p><b>Step 5</b>: A composite ranking of all papers will be determined, and each author’s paper ranking will be adjusted based on a measure of the “quality” of the reviews provided by the author.  The adjustment is designed to provide an incentive to all authors to do an honest and thorough job of reviewing the papers to which they are assigned.</p>
          <p><b>Step 6</b>: Final acceptance/rejection decision making will be done by the workshop organizer as currently done.</p>
          <p><b>Note: failure to complete this review and ranking within the allotted times will result in the disqualification of the author’s own paper.</b></p>
          <p>Anonymity of reviewers will be preserved as authors will not know which of the other authors review their papers.</p>
          <p>The advantages of adopting the new peer-review system are the following:</p>
          <p>1. Your papers will not be reviewed by someone who does not have sufficient expertise in the topic area. <br/>
          2. As you will commit significant time and efforts in reviewing the papers assigned to you, you would expect other authors who review your papers more committed than the reviewers of the current system.<br/>
          3. You have the opportunity to clarify any misunderstanding the reviewers may have on your paper through interactive discussions. Such communications are expected to improve the overall review quality.<br/>
          4. Potential review bias will be minimized since biases will have serious consequences, which is lacking in current review system.<br/>
          5. The overall review process is usually much faster than current review system.<br/>
          </p>
</p> -->
<!--         <h3>Review Process of Pevals</h3>
        <p>Introduction to the new peer-review system – Pevals (Participant-engaged evaluation system)</p>
        <p>The review process consists of the following steps:</p>
        <p><b>Step 1:</b> Each contact author will claim organizational or individual conflicts-of-interest with all the other submissions. (10/1/2017 - 10/2/2017)</p>
        <p><b>Step 2:</b> Each contact author will be assigned to review and rank three other papers. Review assignments will be made so as to avoid organizational or individual conflicts-of-interest. All authors must complete their review and ranking of the assigned papers within 10 days of the date of their assignment.  (10/3/2017 - 10/13/2017)</p>
        <p><b>Step 3:</b> A composite ranking of all papers will be determined, and each author’s paper ranking will be adjusted based on a measure of the “quality” of the reviews provided by the author. The adjustment is designed to provide an incentive to all authors to do an honest and thorough job of reviewing the papers to which they are assigned.</p>
        <p><b>Step 4:</b> Final acceptance/rejection decision making will be done by the workshop organizer based on the review results in both Pevals and Cyberchair (BIBM 2017 official review system).</p>
        <p><b>Note: failure to complete this review and ranking within the allotted times will result in the disqualification of the author’s own paper.</b></p>
        <p>Anonymity of reviewers will be preserved as authors will not know which of the other authors review their papers.</p>
        <p>The advantages of adopting the new peer-review system are the following:</p>
        <p>1. As you will commit significant time and efforts in reviewing the papers assigned to you, you would expect other authors who review your papers more committed than the reviewers of the current system.</p>
        <p>2. Potential review bias will be minimized since biases will have serious consequences, which is lacking in current review system.</p> -->
        </div>
      </div>
    </section>

    <section id="program">
      <div class="row">
        <div class="span12">
          <h3>Program</h3>
          <p>TBD</p>
          <!--<p>The workshop is scheduled in the morning of 12/15/2016 from 8:00 am to 12:00 pm.  The conference program of  BIBM 2016 is now available at <a href="https://cci.drexel.edu/ieeebibm/bibm2016/BIBM2016Program.pdf">https://cci.drexel.edu/ieeebibm/bibm2016/BIBM2016Program.pdf</a>.</p>-->
        </div>
      </div>
    </section>

    <section id="papers">
      <div class="row">
        <div class="span12">
          <h3>Accepted Papers</h3>
          <ul>
            <!-- <li>Rebecca Lin, Muhammad Amith, Chen Liang, and Cui Tao, "Designing an Ontology for Emotion-driven Visual Representations"</li>
            <li>Rui Zhang, Sisi Ma, Liesa Shanahan, Jessica Munroe, Sarah Horn, and Stuart Speedie, "Automatic Methods to Extract New York Heart Association Classification from Clinical Notes"</li>
            <li>Zhiwei Chen, Zhe He, Xiuwen Liu, and Jiang Bian, "An Exploration of Semantic Relations in Neural Word Embeddings Using Extrinsic Knowledge"</li>
            <li>Rashmie Abeysinghe and Licong Cui, "Query-constraint-based Association Rule Mining from Diverse Clinical Datasets in the National Sleep Research Resource"</li>
            <li>Biyang Yu and Zhe He, "Exploratory Textual Analysis of Consumer Health Languages for People who are D/deaf and Hard of Hearing"</li>
            <li>Yadan Fan, Lu He, and Rui Zhang, "Evaluating Automatic Methods to Extract Patients’ Supplement Use from Clinical Reports"</li>
            <li>Hee-Jin Lee, Yaoyun Zhang, Min Jiang, Jun Xu, Cui Tao, and Hua Xu, "Towards practical temporal relation extraction from clinical notes: an analysis of direct temporal relations"</li>
            <li>Juan Antonio Lossio-Ventura, William Hogan, Francois Modave, Yi Guo, Zhe He, Amanda Hicks, and Jiang Bian, "OC-2-KB: A software pipeline to build an evidence-based obesity and cancer knowledge base"</li>
            <li>Hansi Zhang, Yi Guo, Qian Li, Thomas George, Elizabeth Shenkman, and Jiang Bian, "Ontology-guided Semantic Data Integration to Support Integrative Data Analysis: A Case Study of Cancer Survival"</li>
            <li>Xinyuan Zhang, Shiqi Wang, Jie Liu, and Cui Tao, "Computer-aided Diagnosis of Four Common Cutaneous Diseases Using Deep Learning Algorithm"</li>
            <li>Huiqin Chen, Dihua Zhang, Guoping Zhang, Xiaofeng Li, Ying Liang, Mohan Kasukurthi, Glen Borchert, and Jingshan Huang, "MeSH term-based semantic analysis of microRNA regulation on glucocorticoid resistance in pediatric acute lymphoblastic leukemia"</li>
            <li>Kaixian Yu, Tingting Zhao, Peixiang Zhao, and Jinfeng Zhang, "Extraction of protein-protein interactions using natural language processing based pattern matching"</li>
            <li>Huawei Jin, Zhenhua Yu, Xiaodan Wang, Weian Chen, Shaolei Guo, Mohan Kasukurthi, Glen Borchert, and Jingshan Huang, "Computational analysis to discover microRNA biomarkers in glioblastoma"</li>
            <li>Jingcheng Du, Yaoyun Zhang, Cui Tao, and Hua Xu, "A pilot study of mining association between psychiatric stressors and symptoms in tweets"</li>
            <li>Zhe He, Yehoshua Perl, Gai Elhanan, Yan Chen, James Geller, and Jiang Bian, "Auditing the Assignments of Top-Level Semantic Types in the UMLS Semantic Network to UMLS Concepts"</li>
            <li>Haodi Li, Qingcai Chen, Buzhou Tang, and Xiaolong Wang, "Chemical-induced Disease Extraction via Convolutional Neural Networks"</li>
            <li>Rashmie Abeysinghe, Eugene W. Hinderer III, Hunter N.B. Moseley, and Licong Cui, "Auditing Subtype Inconsistencies among Gene Ontology Concepts"</li> -->
          </ul>
          <!-- <ul><h4>Full Papers</h4>
            <li>Yadan Fan, Lu He, and Rui Zhang. <b><i>Classification of Status for Supplement Use in Clinical Notes</i></b>.</li>
            <li>Jingshan Huang, Bi Liu, Yang Liu, and Ji Chen. <b><i>The utilization of the OmniSearch semantic search tool to explore various microRNA regulation mechanisms in osteoarthritis</i></b>.</li>
            <li>Jingshan Huang, Dejing Dou, Jun She, Andrew Limper, Yanan Yang, and Ping Yang. <b><i>A comprehensive (biological and computational) investigation on the role of microRNA::mRNA regulations performed in chronic obstructive pulmonary disease and lung cancer</i></b></li>
            <li>Wenxin Ning and Ming Yu. <b><i>Exploiting Distributional Semantics to Benefit Machine Learning in Automated Clinical Text Classification: An Initial Study</i></b>.</li>
            <li>Juan Antonio Lossio-Ventura, William Hogan, François Modave, Amanda Hicks, Josh Hanna, Yi Guo, Zhe He, and Jiang Bian. <b><i>Towards an obesity-cancer knowledge base: biomedical entity identification and relation detection</i></b>.</li>
            <li>Shengyu Liu, Kai Chen, Qingcai Chen, and Buzhou Tang. <b><i>Dependency-based convolutional neural network for drug-drug interaction extraction</i></b>.</li>
            <li>Mohcine Madkour, Hsing-yi Song, Jingcheng Du, and Cui Tao. <b><i>A Representational Analysis of Temporal Uncertainty in Clinical Events</i></b>.</li>
          </ul>
          <ul><h4>Short Papers</h4>
            <li>Lawrence Chan, Cesar Wong, and Keith Chiu. <b><i>Ontological Features of Electronic Health Records Reveal Distinct Association Patterns in Hepatocellular Carcinoma and Liver Metastasis</i></b>.</li>
          </ul> -->
        </div>
      </div>
    </section>

    <section id="registration">
      <div class="row">
        <div class="span12">
          <h3>Registration</h3>
          <p>At least one author of an accepted paper needs to register as one full registration in order to have the paper published in the conference proceedings. </p>
          <!--The early registration deadline is <b>November 20, 2016</b>.</p>
          <p>The URL of the registration is at <a href="http://www.cvent.com/events/2016-ieee-international-conference-on-bioinformatics-and-biomedicine-bibm-/event-summary-e7475f7d5ae2445b926a02ff88924fed.aspx" target="_blank">http://www.cvent.com/events/2016-ieee-international-conference-on-bioinformatics-and-biomedicine-bibm-/event-summary-e7475f7d5ae2445b926a02ff88924fed.aspx</a>.</p>-->
        </div>
      </div>
    </section>

    <section id="contact">
      <div class="row">
        <div class="span12">
          <h3>Contact</h3>
          <p>Please email Workshop co-chair, <a href="mailto:Zhe.He@cci.fsu.edu" target="_blank">Zhe He (Zhe.He@cci.fsu.edu)</a></p>
        </div>
      </div>
    </section>

    <section id="history">
      <div class="row">
        <div class="span12">
          <h3>Past Workshops</h3>
          <ul>
            <li><a href="/sepda2017.html" target="_blank">The 2nd International Workshop on Semantics-Powered Data Analytics (SEPDA 2017)</a>, in conjunction with the <a href="http://muii.missouri.edu/bibm2017/" target="_blank">IEEE International Conference on Bioinformatics and Biomedicine</a> (BIBM 2017), Kansas City, MO, USA, November 13, 2017</li>
            <li><a href="/sepda2016.html" target="_blank">The 1st International Workshop on Semantics-Powered Data Analytics (SEPDA 2016)</a>, in conjunction with the IEEE International Conference on Bioinformatics and Biomedicine (BIBM 2016), Shenzhen, China, December 15, 2016</li>
          </ul>
        </div>
      </div>
    </section>

            <hr>

            <footer>
                <p>&copy; Semantic-Powered.org 2018</p>
            </footer>

        </div> <!-- /container -->

        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.9.1.min.js"><\/script>')</script>

        <script src="js/vendor/bootstrap.min.js"></script>

        <script src="js/plugins.js"></script>
        <script src="js/main.js"></script>
        <script>
        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

        ga('create', 'UA-50982144-4', 'auto');
        ga('send', 'pageview');

      </script>
    </body>
</html>
