<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!-->
<html class="no-js">
<!--<![endif]-->

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <title></title>
  <meta name="description" content="">
  <meta name="viewport" content="width=device-width">

  <link rel="stylesheet" href="css/bootstrap.min.css">
  <style>
    body {
      padding-top: 60px;
      padding-bottom: 40px;
    }
  </style>
  <link rel="stylesheet" href="css/bootstrap-responsive.min.css">
  <link rel="stylesheet" href="css/main.css">

  <script src="js/vendor/modernizr-2.6.2-respond-1.1.0.min.js"></script>
</head>

<body class="preview" id="top" data-spy="scroll" data-target=".subnav" data-offset="80">
  <!--[if lt IE 7]>
            <p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
        <![endif]-->

  <!-- This code is taken from http://twitter.github.com/bootstrap/examples/hero.html -->

  <div class="navbar navbar-fixed-top">
    <div class="navbar-inner">
      <div class="container">
        <a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </a>
        <a class="brand" href="#">&nbsp;</a>
        <div class="nav-collapse collapse">
          <ul class="nav">
            <li><a href="#">SEPDA 2019</a></li>
          </ul>
        </div>

      </div>

    </div>
  </div>

  <div class="container">

    <!-- Main hero unit for a primary marketing message or call to action -->
    <!--
            <div class="hero-unit">
                <h1>Hello, world!</h1>
                <p>This is a template for a simple marketing or informational website. It includes a large callout called the hero unit and three supporting pieces of content. Use it as a starting point to create something more unique.</p>
                <p><a class="btn btn-primary btn-large">Learn more &raquo;</a></p>
            </div>
            -->
    <!-- Masthead
================================================== -->
    <header class="jumbotron subhead" id="overview">
      <div class="row">
        <div class="span12">
          <br />
          <h2>The 4th International Workshop on Semantics-Powered Data Mining and Analytics (SEPDA 2019)</h2>
          <br />
          <p class="lead">
            in Conjunction with the 18th International Semantic Web Conference (ISWC 2019)
          </p>
        </div>
        <div class="span6">
          <!--
      <div class="bsa well">
          <div id="bsap_1277971" class="bsarocks bsap_c466df00a3cd5ee8568b5c4983b6bb19"></div>
      </div>
    -->
        </div>
      </div>
      <div class="subnav">
        <ul class="nav nav-pills">
          <li><a href="#overview">Overview</a></li>
          <li><a href="#scope">Topics & Important Dates & Submission</a></li>
          <li><a href="#organization">Organizing</a></li>
          <li><a href="#papers">Accepted Paper</a></li>
          <li><a href="#proceedings">CEUR Proceedings</a></li>
          <li><a href="#schedule">Workshop Schedule</a></li>
          <li><a href="#history">Past Workshops</a></li>
        </ul>
      </div>
    </header>

    <section id="overview">
      <div class="row">
        <div class="span12">
          <h3>Overview</h3>
          <p>Biomedical ontologies and controlled terminologies provide structured domain knowledge to a variety of
            health information systems. The rich thesaurus with concepts linked by semantic relationships has been
            widely used in natural language processing, data mining, machine learning, semantic annotation, and
            automated reasoning. The dramatically increasing amount of health-related data poses unprecedented
            opportunities for mining previously unknown knowledge with semantics-powered data mining and analytics
            methods. However, due to the heterogeneity of different data sources, it is a challenging problem to exploit
            multiple sources to solve real-world problems such as designing cost-effective treatment plan for patients,
            designing generalizable clinical trials, drug repurposing, and clinical phenotyping. The goal of this
            workshop is to bring people in the field of ontologies, data mining, knowledge representation, knowledge
            management, and data analytics to discuss innovative semantic methods, applications, and data analytics to
            address problems in healthcare, biomedicine, public health, and clinical research with biomedical, clinical,
            behavioral, and social web data.</p>


          <p>In the past three years, SEPDA has been established as a key venue for disseminating research on health
            data analytics using semantic web technologies such as ontologies. In the past few years, we have seen an
            increasing interest in using semantic web technologies for health data analysis with more and more
            submissions that present novel methods and applications for linked open data, information extraction,
            semantic-web-based knowledge bases, and deep learning. The NIH Data Science Strategic Plan released in June
            2018 explicitly commits to ensuring that all data-science activities and products supported by the agency
            adhere to the FAIR principles, meaning that data be Findable, Accessible, Interoperable, and Reusable.
            Semantic web technologies play a crucial role to address the FAIR principles. With the infrastructure
            support such as NCBO’s BioPortal for ontology maintenance, the CEDAR software for metadata creation and
            validation, more and more researchers are using ontologies and semantic web technologies for knowledge
            representation, semantic inference, natural language processing, and data analytics. Meanwhile, we received
            submissions that use semantic-based methods to tackle critical problems in biomedical informatics such as
            extracting drug-drug interaction, drug repurposing, adverse drug reaction, detecting early signals for
            cognitive impairment, and visualizing dietary supplement knowledge. It is thus critical for SEPDA to
            continue our momentum and allow researchers to present and discuss novel methods and applications in this
            fast growing field. This year, we will hold SEPDA in ISWC, the premier conference in semantic web.</p>
        </div>
      </div>
    </section>

    <section id="scope">
      <div class="row">
        <div class="span12">
          <h3>Topics</h3>
          <p>Topics of interest include but not limited to:</p>
          <ul>
            <li><b>Semantics-based Data Mining and Analytics</b>
              <ul>
                <li>Ontology-based text mining and natural language processing</li>
                <li>Semantics-powered data mining and machine learning from biomedical, clinical, or social web data
                </li>
                <li>Information Extraction from biomedical, clinical, or social web data</li>
                <li>Semantic annotation on biomedical, clinical or social web data</li>
              </ul>
            </li>
            <li><b>Ontologies and Controlled Terminologies</b>
              <ul>
                <li>Ontology development and enrichment</li>
                <li>Quality assurance of ontologies and controlled terminologies</li>
                <li>Semantic harmonization and ontology alignment</li>
                <li>Knowledge representation and reasoning</li>
              </ul>
            </li>
            <li><b>Data Integration</b>
              <ul>
                <li>Linked open data</li>
                <li>Novel approaches for data integration of heterogenous data sources</li>
                <li>Large scale data integration</li>
              </ul>
            </li>
            <li><b>Application</b>
              <ul>
                <li>Novel tools and ontologies for data interpretation and visualization</li>
                <li>Pharmacovigilance</li>
                <li>Drug repurposing</li>
                <li>Clinical trial generalizability assessment using ontologies</li>
                <li>Algorithmic phenotyping and cohort identification using ontologies</li>
                <li>Improving the literacy of health information consumers</li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
      <div class="row">
        <div class="span12">
          <h3>Important Dates</h3>
          <p>June 28, 2019: Due date for full / work-in-progress workshop papers submission</p>
          <p>July 24, 2019: Notification of paper acceptance to authors</p>
          <p>August 10, 2019: Camera-ready of accepted papers</p>
          <p>October 26, 2019: SEPDA 2019 Workshop</p>
        </div>
      </div>
      <div class="row">
        <div class="span12">
          <h3>Submission</h3>
          <p>We are inviting original research submissions (FULL 12 pages), work-in-progress (SHORT 4 pages), and poster
            abstracts (2 pages).</p>
          <p>All the accepted submissions will be presented in SEPDA 2019. The abstracts of full papers (2 pages) and
            the short papers (4 pages) will be published in CEUR Workshop Proceedings.</p>
          <p>The full papers will be published in a mainstream journal in semantic web / biomedical informatics (TBA).
          </p>
          <p>Submission site: <a href="https://easychair.org/conferences/?conf=sepda2019" target="_blank">Submit a New
              Paper</a></p>
          <p>Submissions must be in English formatted in the style of the Springer Publications format for Lecture Notes
            in Computer Science (LNCS). For details on the LNCS style, see <a
              href="https://www.springer.com/us/computer-science/lncs/conference-proceedings-guidelines"
              target="_blank">Springer’s Author Instructions</a>.</p>
        </div>
      </div>
    </section>


    <section id="organization">
      <div class="row">
        <div class="span12">
          <h3>Organizing</h3>
          <ul>
            <li><b>Organizing Committee</b>
              <ul>
                <li>Zhe He, PhD, School of Information, Florida State University, USA</li>
                <li>Cui Tao, PhD, School of Biomedical Informatics, University of Texas Health Science Center at
                  Houston, USA</li>
                <li>Jiang Bian, PhD, Biomedical Informatics, Health Outcomes & Policy, College of Medicine, University
                  of Florida, USA</li>
                <li>Rui Zhang, PhD, Institute for Health Informatics, University of Minnesota, USA</li>
              </ul>
            </li>
            <li><b>Program Committee Members</b>
              <ul>
                <li>James Cimino, PhD, University of Alabama, Birmingham, USA</li>
                <li>Yi Guo, PhD, University of Florida, USA</li>
                <li>Xia Jing, PhD, Ohio University, USA</li>
                <li>Jake Luo, PhD, University of Wisconsin–Milwaukee, USA</li>
                <li>Yonghui Wu, PhD, University of Florida, USA</li>
                <li>Licong Cui, PhD, School of Biomedical Informatics, University of Texas Health Science Center at
                  Houston, USA</li>
                <li>Feichen Shen, PhD, Mayo Clinic, USA</li>
                <li>Fleur Mougin, PhD, University of Bordeaux, USA</li>
                <li>Yuji Zhang, University of Maryland School of Medicine, USA</li>
                <li>Zhengxing Huang, Zhejiang University, China</li>
                <li>Guoqian Jiang, Mayo Clinic College of Medicine, USA</li>
              </ul>
            </li>
            <li>
              <b>Student Program Committee Members</b>
              <ul>
                <li>Jingcheng Du, University of Texas Health Science Center at Houston, USA</li>
                <li>Hansi Zhang, University of Florida, USA</li>
                <li>Yunpeng Zhao, University of Florida, USA</li>
                <li>Seyedeh Neelufar Payrovnaziri, Florida State University, USA</li>
                <li>Xi Yang, University of Florida, USA</li>
                <li>Xing He, University of Florida, USA</li>
                <li>Xinyuan Zhang, University of Texas Health Science Center at Houston, USA</li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </section>

    <!-- <section id="review">
      <div class="row">
        <div class="span12">
        <h3>Review Process of Pevals</h3>
        <p>Introduction to the new peer-review system – Pevals (Participant-engaged evaluation system)</p>
        <p>The review process consists of the following steps:</p>
        <p><b>Step 1:</b> Each contact author will claim organizational or individual conflicts-of-interest with all the other submissions. (10/1/2017 - 10/2/2017)</p>
        <p><b>Step 2:</b> Each contact author will be assigned to review and rank three other papers. Review assignments will be made so as to avoid organizational or individual conflicts-of-interest. All authors must complete their review and ranking of the assigned papers within 10 days of the date of their assignment.  (10/3/2017 - 10/13/2017)</p>
        <p><b>Step 3:</b> A composite ranking of all papers will be determined, and each author’s paper ranking will be adjusted based on a measure of the “quality” of the reviews provided by the author. The adjustment is designed to provide an incentive to all authors to do an honest and thorough job of reviewing the papers to which they are assigned.</p>
        <p><b>Step 4:</b> Final acceptance/rejection decision making will be done by the workshop organizer based on the review results in both Pevals and Cyberchair (BIBM 2017 official review system).</p>
        <p><b>Note: failure to complete this review and ranking within the allotted times will result in the disqualification of the author’s own paper.</b></p>
        <p>Anonymity of reviewers will be preserved as authors will not know which of the other authors review their papers.</p>
        <p>The advantages of adopting the new peer-review system are the following:</p>
        <p>1. As you will commit significant time and efforts in reviewing the papers assigned to you, you would expect other authors who review your papers more committed than the reviewers of the current system.</p>
        <p>2. Potential review bias will be minimized since biases will have serious consequences, which is lacking in current review system.</p>
        </div>
      </div>
    </section> -->


    <section id="papers">
      <div class="row">
        <div class="span12">
          <h3>Accepted Papers</h3>
          <ul>
            <li>Nur Hafieza Ismail, Mengnan Du, Ninghao Liu, Zhe He and Xia Hu. Using Deep Neural Network to Identify
              Cancer Survivors Living with Post-Traumatic Stress Disorder on Social Media</li>

            <li>Gilles Vandewiele, Bram Steenwinckel, Femke Ongenae and Filip De Turck. Inducing a decision tree with
              discriminative paths to classify entities in a knowledge graph</li>

            <li>Jiebin Chu, Wei Dong and Zhengxing Huang. Treatment Effect Prediction with Generative Adversarial
              Networks Using Electronic Health Records</li>

            <li>Hansi Zhang, Xing He, Tyler Harrison and Bian Jiang. Aero: An Evidence-based Semantic Web Knowledge Base
              of Cancer Behavioral Risk Factors</li>

            <li>Muhammad Amith, Rebecca Lin, Licong Cui, Dennis Wang, Anna Zhu, Grace Xiong, Hua Xu, Kirk Roberts and
              Cui Tao. An Ontology-Powered Dialogue Engine For Patient Communication of Vaccines</li>

            <li>Hansi Zhang, Yi Guo and Jiang Bian. Ontology for Documentation of Variable and Data Source Selection
              Process to Support Integrative Data Analysis in Cancer Outcomes Research</li>

            <li>Alyssa Lees, Chris Welty, Jacek Korycki and Shubin Zhao. Taxonomy Embeddings on PubMed Article Subject
              Headings</li>

            <li>Guocai Chen, Xiaoqian Jiang and Wenjin Zheng. Feature compression for predicting effective drug
              combination</li>

            <li>Guozheng Rao, Wenwen Li, Xiaowang Zhang, Li Zhang and Cui Tao. A multi-source linked open data fusion
              method for gene-disorder-drug relationship mining</li>

            <li>Anderson Rossanez and Julio Cesar dos Reis. Generating Knowledge Graphs from Scientific Literature of
              Degenerative Diseases</li>

            <li>Keyuan Jiang, Tingyu Chen, Liyuan Huang, Gelareh Karbaschi and Gordon Bernard. Semantic Similarity of
              Medication-Effect Relations Inferred from Neural Embedding</li>

          </ul>
        </div>
      </div>
    </section>

    <section id="proceedings">
      <div class="row">
        <div class="span12">
          <h3>CEUR Proceedings</h3>
          <p>Links: <a href="http://ceur-ws.org/Vol-2427/" target="_blank">http://ceur-ws.org/Vol-2427/</a></p>
        </div>
      </div>
    </section>

    <section id="schedule">
      <div class="row">
        <div class="span12">
          <h3>Workshop Schedule</h3>
          <table style="border-collapse:collapse;border-spacing:0; width: 100%;" class="tg table">
            <colgroup>
              <col style="width: 147px">
              <col style="width: 427px">
              <col style="width: 380px">
            </colgroup>
            <tr>
              <th
                style="font-family:Arial, sans-serif;font-size:20px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;background-color:#c0c0c0;text-align:center;vertical-align:middle"
                colspan="3"><span style="font-weight:bold">The 4th International Workshop on Semantics-Powered Data
                  Analytics</span><br>Date, Time: October 27, 2019, 9:20 am – 5:20 pm; Room: To Be
                Determined<br>Workshop Chairs: Zhe He, Cui Tao, Jiang Bian, and Rui Zhang</th>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:18px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:left;vertical-align:middle">
                Time<br></td>
              <td
                style="font-family:Arial, sans-serif;font-size:18px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:left;vertical-align:middle">
                Title</td>
              <td
                style="font-family:Arial, sans-serif;font-size:18px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:left;vertical-align:top">
                Presenter/Author</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                9:20 am - 9:25 am</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:center;vertical-align:middle"
                colspan="2">Open Remarks</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                9:25 am - 10:40 am</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:center;vertical-align:middle"
                colspan="2">Part 1</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                Inducing a decision tree with discriminative paths to classify entities in a knowledge graph</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Gilles Vandewiele (University Ghent)</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                Using Deep Neural Network to Identify Cancer Survivors Living with Post-Traumatic Stress Disorder on
                Social Media</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Nur Hafieza Ismail (Texas A&M University) (Remote Presentation)</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Semantic Similarity of Medication-Effect Relations Inferred from Neural Embedding</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Keyuan Jiang (Purdue University)</td>
            </tr>


            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                10:40 am - 11:20 am</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:center;vertical-align:middle"
                colspan="2">Break</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                11:20 am - 12:40 pm</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:center;vertical-align:middle"
                colspan="2">Part 2</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                Taxonomy Embeddings on PubMed Article Subject Headings</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Shubin Zhao (Google Research)</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                Ontology for Documentation of Variable and Data Source Selection Process to Support Integrative Data
                Analysis in Cancer Outcomes Research</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Hansi Zhang (University of Florida) (Remote Presentation)</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                An Ontology-Powered Dialogue Engine For Patient Communication of Vaccines</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Muhammad Amith (University of Texas Health Science Center)</td>
            </tr>

            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                12:40 pm - 14:00 pm</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:center;vertical-align:middle"
                colspan="2">Lunch</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                14:00 pm - 15:20 pm</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:center;vertical-align:middle"
                colspan="2">Part 3</td>
            </tr>

            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                Feature compression for predicting effective drug combination</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Wenjin Zheng (University of Texas Health Science Center)</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                Aero: An Evidence-based Semantic Web Knowledge Base of Cancer Behavioral Risk Factors</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Hansi Zhang (University of Florida) (Remote Presentation)</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                A multi-source linked open data fusion method for gene-disorder-drug relationship mining</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Guozheng Rao</td>
            </tr>

            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                15:20 pm - 16:00 pm</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:center;vertical-align:middle"
                colspan="2">Break</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                16:00 pm - 16:50 pm</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:center;vertical-align:middle"
                colspan="2">Part 4</td>
            </tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                Treatment Effect Prediction with Generative Adversarial Networks Using Electronic Health Records</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Zhengxing Huang (Remote Presentation)</td>
            </tr>

            <tr>
            <tr>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
              </td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:middle">
                Generating Knowledge Graphs from Scientific Literature of Degenerative Diseases</td>
              <td
                style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
                Anderson Rossanez</td>
            </tr>

            <td
              style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">
              16:50 pm - 17:20 pm</td>
            <td
              style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;font-weight:bold;text-align:center;vertical-align:top"
              colspan="2">Closing Remarks and Feedback</td>
            </tr>
          </table>
        </div>
      </div>
    </section>

    <!-- <section id="registration">
      <div class="row">
        <div class="span12">
          <h3>Registration</h3>
          <p>At least one author of an accepted paper needs to register as one full registration in order to have the paper published in the conference proceedings. </p>
          The early registration deadline is <b>November 20, 2016</b>.</p>
          <p>The URL of the registration is at <a href="http://www.cvent.com/events/2016-ieee-international-conference-on-bioinformatics-and-biomedicine-bibm-/event-summary-e7475f7d5ae2445b926a02ff88924fed.aspx" target="_blank">http://www.cvent.com/events/2016-ieee-international-conference-on-bioinformatics-and-biomedicine-bibm-/event-summary-e7475f7d5ae2445b926a02ff88924fed.aspx</a>.</p>
        </div>
      </div>
    </section> -->
    <!-- 
    <section id="contact">
      <div class="row">
        <div class="span12">
          <h3>Contact</h3>
          <p>Please email Workshop co-chair, <a href="mailto:Zhe.He@cci.fsu.edu" target="_blank">Zhe He (Zhe.He@cci.fsu.edu)</a></p>
        </div>
      </div>
    </section> -->

    <section id="history">
      <div class="row">
        <div class="span12">
          <h3>Past Workshops</h3>
          <ul>
            <li>The 1st SEPDA workshop (<a href="http://semantics-powered.org/sepda2016.html" target="_blank">SEPDA
                2016</a>) was held in conjunction with IEEE BIBM 2016 in Shenzhen, China. SPEDA 2016 was a half-day
              workshop. Dr. Hua Xu, a leading expert in the field of Biomedical Informatics, gave the keynote speech on
              the bioCADDIE project. Eight quality papers were presented. Selected papers were invited to publish the
              extended version to the special issue on “<a href="https://www.hindawi.com/journals/jhe/si/461480/"
                target="_blank">Semantics-Powered Healthcare Engineering and Data Analytics</a>” in the Journal of
              Healthcare Engineering.</li>
            <li>The 2nd SEPDA Workshop (<a href="http://semantics-powered.org/sepda2017.html" target="_blank">SEPDA
                2017</a>) was held in conjunction with IEEE BIBM 2017 in Kansas City, Missouri, USA. SPEDA 2017 was a
              full day workshop. 17 high quality papers were presented in the workshop. Extended journal version of the
              papers was published in <a
                href="https://bmcmedinformdecismak.biomedcentral.com/articles/supplements/volume-18-supplement-2"
                target="_blank">BMC Medical Informatics & Decision Making</a> (IF: 2.134).</li>
            <li>The 3rd SEPDA Workshop (<a href="http://semantics-powered.org/sepda2018.html" target="_blank">SEPDA
                2018</a>) was held in conjunction with IEEE BIBM 2018 in Madrid, Spain. SPEDA 2018 was a full day
              workshop. 14 high quality papers were presented in the workshop. Extended journal version of the papers
              will be published in <a href="https://bmcmedinformdecismak.biomedcentral.com/" target="_blank">BMC Medical
                Informatics & Decision Making</a> (IF: 2.134).</li>
          </ul>
        </div>
      </div>
    </section>

    <hr>

    <footer>
      <p>&copy; Semantic-Powered.org 2019</p>
    </footer>

  </div> <!-- /container -->

  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
  <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.9.1.min.js"><\/script>')</script>

  <script src="js/vendor/bootstrap.min.js"></script>

  <script src="js/plugins.js"></script>
  <script src="js/main.js"></script>
  <script>
      (function (i, s, o, g, r, a, m) {
        i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
          (i[r].q = i[r].q || []).push(arguments)
        }, i[r].l = 1 * new Date(); a = s.createElement(o),
          m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
      })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');

    ga('create', 'UA-50982144-4', 'auto');
    ga('send', 'pageview');

  </script>
</body>

</html>